.github/workflows/jacktrip.yml

```diff
@@ -98,7 +99,12 @@ jobs:
         if: runner.os == 'Linux'
         run: |
           sudo apt-get update
-          sudo apt-get install --yes libjack-dev qt5-default qt5-qmake qttools5-dev
+          sudo apt-get install --yes libjack-dev
+          if [[ -n "${{ matrix.static-qt-version }}" ]]; then 
+            sudo apt-get install --yes libfreetype6-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev libx11-xcb-dev libdrm-dev libglu1-mesa-dev libwayland-dev libwayland-egl1-mesa libgles2-mesa-dev libwayland-server0 libwayland-egl-backend-dev
```

Comment:
Add all those listed packages here?

Reply:
I think all the needed packages are included - at some point working on this I removed the ones that said they are already installed.

Just in case triggering a new build installing libxcb explicitly.

Reply:
Could you add those?

```
    libfontconfig1-dev
    libfreetype6-dev
    libx11-dev
    libx11-xcb-dev
    libxext-dev
    libxfixes-dev
    libxi-dev
    libxrender-dev
    libxcb1-dev
    libxcb-glx0-dev
    libxcb-keysyms1-dev
    libxcb-image0-dev
    libxcb-shm0-dev
    libxcb-icccm4-dev
    libxcb-sync0-dev
    libxcb-xfixes0-dev
    libxcb-shape0-dev
    libxcb-randr0-dev
    libxcb-render-util0-dev
    libxcd-xinerama-dev
    libxkbcommon-dev
    libxkbcommon-x11-dev
```

Reply:
> I think all the needed packages are included - at some point working on this I removed from `apt install` the ones that said they are already installed.
> 
> Just in case triggering a new build installing libxcb explicitly.

Reply:
It's libxcb-xinerama-dev and not libxcd-xinerama-dev. I just copy and pasted.